main.py

```diff
@@ -17,6 +13,19 @@
 screen = pygame.display.set_mode((screen_width, screen_height))
 
 
-state = State(screen, args.keyboard)
-state.playIntro()
-state.playGame()
\ No newline at end of file
+state = StateMachine()
+
+
+while state.state != StateCode.END:
+
+    if state.state == StateCode.MENU:
+        state.state = StateMachine.playMenu(screen)
+    elif state.state == StateCode.INTRO:
+        state.state = StateMachine.playIntro(screen)
+    elif state.state == StateCode.PLAYING:
+        state.state = StateMachine.playGame(screen, args.keyboard)
+    else:
+        raise EnvironmentError
+
+
+pygame.quit()
```

Comment:
I reckon if we just put this at the start of the code, it'll complete much quicker.